js/apps/account-ui/src/root/Header.tsx

```diff
@@ -51,9 +51,7 @@ export const Header = () => {
       features={{ hasManageAccount: false }}
       brand={{
         href: indexHref,
-        src: brandImage.startsWith("/")
-          ? joinPath(environment.resourceUrl, brandImage)
-          : brandImage,
+        src: joinPath(environment.resourceUrl, brandImage),
```

Comment:
Would this not break if the image that is loaded is coming from another URL as configured in the theme? E.g. `http://some-other-domain/logo.png`